Automattic/jetpack
# User management: Add a checkbox for stating if the user is a contractor

Context: p1710910648561939-slack-C06DN6QQVAQ

## Proposed changes:


The user creation, with SSO enabled, now offers the possibilty of stating whether a new user is a contractor.

![image](https://github.com/Automattic/jetpack/assets/52076348/19c5fdc1-975e-4c4c-aaed-9f6e649853d4)


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Checkout branch
* Sync to atomic site
* Add new user in wp-admin and accept invitation (with external user checkbox checked/unchecked)
* Verify the `contractor` label appears is added to the user after the invitation is accepted (`people/team/{SITE_SLUG}`)
<img width="632" alt="CleanShot 2024-03-20 at 23 32 29@2x" src="https://github.com/Automattic/jetpack/assets/10482592/e0f977c3-cc39-4ba8-a902-ab81508a10cf">
* Verify there are no errors when you delete the user
